roles/ffh.mesh_wireguard/templates/ferm.conf.j2

```diff
@@ -0,0 +1,15 @@
+domain (ip6) {
+  table filter {
+    chain INPUT {
+      interface (wgtest-10 wgtest-11 wgtest-12 wgtest-13 wgtest-14 wgtest-15 wgtest-16 wgtest-17 wgtest-18 wgtest-19 wgtest-20 wgtest-21 wgtest-22 wgtest-23 wgtest-99) proto udp dport 4789 ACCEPT;
+    }
+  }
+}
+domain (ip ip6) {
+  table filter {
+    chain INPUT {
```

Comment:
With recent changes on master (https://github.com/freifunkh/ansible/commit/4c816c464e80132e06cfbf9a5f886700ebeb2878), you can change this to:
  ```
    chain INPUT_SOURCE_IS_NOT_FFH_EXIT {
      udp dport 51810:51823 ACCEPT;
      proto udp dport 51899 ACCEPT;
    }
```
And you will block freifunk in freifunk stuff :)

Reply:
Do not forget to roll the new commit on sn07, as I haven't done this yet. But I left you a hint in the commit message about which roles to roll. :)

Reply:
've been there, done that.
Rolled every role.